externs/shaka/player.js

```diff
@@ -97,6 +98,9 @@ shaka.extern.StateChange;
  * @property {number} droppedFrames
  *   The total number of frames dropped by the Player.  This may be
  *   <code>NaN</code> if this is not supported by the browser.
+ * @property {number} corruptedFrames
+ *   The total number of frames corrupted by the browser.  This may be
```

Comment:
I think it is also wrong to say the browser corrupted the frames.  Maybe this is a subtlety of English, but this means that the frames were okay, then the browser changed them in a way that broke something.

Instead, I would suggest "number of corrupted frames dropped by the browser".  The frames were already corrupt when they got here, and the browser was forced to discard them because they could not be properly decoded.

Reply:
My native language is Spanish and I have misunderstood what you wanted to express to me, I'm sorry. I have changed the phrase to express the right thing.